### PR TITLE
Update grays for repo-list element

### PR DESCRIPTION
**Pull request by nataliafitzgerald**
Grays for the repo-list element on DM pages are not updated to our latest color palette naming convention and do not have sufficient contrast. Additionally we pull in a type style that isn't in our type hierarchy for the "p" text under the heading. I went ahead and changed this to 1em. 

## Changes
- Change background color to Gray 10
- Change text color to Gray
- Change p to 16px (1em) 

## Testing
- http://nataliafitzgerald.github.io/design-manual/page-components/form-fields.html

## Review
- @cfarm 
[Preview this PR without the whitespace changes](?w=0)

## Screenshots

**Before**
screen shot 2017-08-23 at 6 07 53 pm

**After**
screen shot 2017-08-23 at 6 07 47 pm

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by nataliafitzgerald** [comment]
@cfarm  - One remaining change I'd like to make is to vertically center the text and icon within the gray box. Currently the text and icon sit a bit too low in the box. I poked around at doing this but couldn't find an ideal way to fix it. Can you make any suggestions?

File: page-components/form-fields.md [hunk 1]
```diff
@@ -672,6 +672,58 @@ To optimize screen reader accessibility, lay out checkboxes vertically rather th
 {:/nomarkdown}
 
 </div>
+<br>
+### Nested checkboxes
```
Comment (nataliafitzgerald) [inline page-components/form-fields.md]: @cfarm - I thought this addition was part of what was merged this morning. It's already on the live page. Do you know why it's showing up here?

**Comment by nataliafitzgerald** [comment]
@cfarm - A lot of what you see here for form-fields updates were me trying to fix conflicts that came up when I tried to merge the updates to the repo-list styling. I'll probably need a refresher on how to do that efficiently. ;) 

It would also probably be ideal for me to start working on specific branches when I work on different pages so that my PR's can be properly focused. I'll need a refresh on how to work that way.

**Comment by cfarm** [comment]
Works as advertised!

**Pull request merged**